woocommerce/woocommerce-ios
# [Woo POS] M2: Add modal banner for simple products only

The PR adds a new modal view, `SimpleProductsModalView`, that provides details about only simple products being supported by Woo POS.

The new view may be triggered from the existing header banner that states "Showing simple products only" when POS mode is entered (see #13317).

Closing the header banner causes an info icon to be shown in the Products header. Tapping that will display the new modal banner for simple products.

The PR also centralizes modal presentation logic in PointOfSaleDashboardViewModel (commit 1f4084b3e9a1ea31096b73ec121555ca54e2013c).

The new view doesn't yet match the design in a pixel-perfect manner per prior agreement this week. The link on the modal view to "Create an order in store management" also doesn't yet work. These can be taken up under a separate ticket.



Closes: #13357



## Testing information

- Enter POS mode.
- Observe the "Showing simple products only" banner is shown.
- Tap the banner or the banner's info icon.
-  Observe the modal is show with "Why can't I see my products?" as the title.
- Click OK or the X icon to close the modal.
- Click the X icon in the header banner.
- Observe an info icon appears in the Products header.
- Tap the info icon.
- Observe the "Why can't I see my products?" modal is shown.

## Screenshots


https://github.com/user-attachments/assets/3edd4439-b633-46ee-aa50-c4e2e5703f95





---
